deployments/ansible/roles/signalfx-agent/tasks/debian_repo.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Add an Apt signing key for Signalfx Agent
+  apt_key:
+    url: "{{ repo_base_url }}/debian.gpg"
+    keyring: /etc/apt/trusted.gpg.d/signalfx.gpg
+    state: present
+
+- name: Add Signalfx Agent repository into sources list
+  apt_repository:
+    repo: "deb {{ repo_base_url }}/debs/signalfx-agent/{{ package_stage }} /"
+    filename: 'signalfx-agent'
+    mode: 644
+    state: present
+
+- name: Clean up the existing Signalfx Agent
```

Comment:
This would result in the agent being uninstalled and reinstalled every time ansible is run on a server right?

Reply:
Updated the task with `force: yes` option as the `downgrade` option is not yet supported for *apt* module